app/templates/courses/classroom-settings-modal.coco.pug

```diff
@@ -74,6 +74,15 @@ block modal-body-content
         option(value="javascript") JavaScript
         if view.enableCpp
           option(value="cpp") C++
+  
+    .form-group
+      label(data-i18n="courses.classroom_live_completion") live completion
+      - var aceConfig = view.classroom.get('aceConfig') || {}
+      - var liveCompletion = aceConfig.liveCompletion
+      if liveCompletion === undefined
+        - liveCompletion = true
+      input#liveCompletion(name="liveCompletion" checked=liveCompletion type='checkbox')
```

Comment:
bootstrap from control makes the checkbox in single line 😂  
oh but i can take a look how to make the input in the same line with label. it could i think

Reply:
okay since we're using bootstrap 3 and it doesn't provide good checkbox styles 😂